app/jobs/acceptance_job.rb

```diff
@@ -0,0 +1,11 @@
+class AcceptanceJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(student_application_id)
+    puts "Yay performed job"
```

Comment:
But if you think this should never make it into the master branch thats fine too.